api/internal/localizer/localizer.go

```diff
@@ -38,8 +39,7 @@ type Localizer struct {
 func NewLocalizer(ldr *LocLoader, validator ifc.Validator, rFactory *resmap.Factory, pLdr *pLdr.Loader) (*Localizer, error) {
 	toDst, err := filepath.Rel(ldr.args.Scope.String(), ldr.Root())
 	if err != nil {
-		log.Fatalf("cannot find path from directory %q to %q inside directory: %s", ldr.args.Scope.String(),
-			ldr.Root(), err.Error())
+		log.Fatalf("cannot find path from %q to child directory %q: %s", ldr.args.Scope.String(), ldr.Root(), err.Error())
```

Comment:
nit: 
- `ldr.args.Scope.String()` --> `ldr.args.Scope`
- `err.Error()` -> `err`

Reply:
Thank you for this comment!! I didn't know that the `fmt` package did this: https://pkg.go.dev/fmt#hdr-Printing